RELEASE.md

```diff
@@ -0,0 +1,56 @@
+# One App release process
+
+This defines the One App release process. 
+
+Releases can be triggered in two ways:
+
+  - [Automated release process](#automated-release-process)
+  - [Manual release process](#manual-release-process)
+
+## Automated release process
+
+ 1. An automated pull request will be raised every Wednesday at 16:00 UTC, this would be from the `prerelease` branch to master. This uses this [pull_request_release workflow](.github/workflows/pull_request_release.yml)and updates [package.json](package.json), [package-lock.json](package-lock.json), [one-app-statics package.json](one-app-statics/package.json) and the changelog(Behind the scene it uses [standard-version](https://github.com/conventional-changelog/standard-version) to generate and update this files) with the changes to be released. The commit message contains the version to be released. Updated this pull request to remove or add any new changes.
+ 2. Once a pull request is reviewed merge the pull request and please ensure that the commit message is updated to follow this pattern  
+
+   ``` bash
+   #chore(release): v5.0.0
+   chore(release): v<semantic-version>
+
+   ```
+
+   > Integration tests will continue to run in Travis as they currently do, after the automated pull request is created it would run the tests on `prerelease` branch.
+
+1. The merge will trigger the automatic generation of a new tag using the semantic version provided during the merging of the pull request above.
+
+2. After the the generated tag is pushed to the branch this will trigger the docker build and push of the images and statics to the docker hub. The development and production images would be accessible in docker [https://hub.docker.com/u/oneamex](https://hub.docker.com/u/oneamex)
+
+3. For the github release notes we are currently using [https://github.com/release-drafter/release-drafter](https://github.com/release-drafter/release-drafter) to generate release notes. Please use the labels specified within [release-drafter](.github/release-drafter.yml) to categorize the different pull requests by adding the labels to them. Update the draft release notes and tie it to the released tag above you can also link this to different artifacts.The statics assets will be published and added to a tag that has been released.
```

Comment:
```suggestion
3. We are currently using [https://github.com/release-drafter/release-drafter](https://github.com/release-drafter/release-drafter) to generate release notes. Please add the labels specified within [release-drafter](.github/release-drafter.yml) to categorize different pull requests. Update the draft release notes and tie it to the release tag above, you can also link this to different artifacts. The statics assets will be published and added to a tag that has been released.
```

Reply:
not sure about this sentence `Update the draft release notes and tie it to the released tag above you can also link this to different artifacts.` is this what is happening or what you need to do ?

Reply:
@JAdshead Currently there is a "draft" release and a "release" created automatically after the docker images and statics are deployed, the "release" contains the statics, so the idea for now was to edit the "release" and copy from the draft all the automatically generated markdown that contains the change-log etc. We could look at an improvement of taking the draft release and use it during the deployment process, but it looks a bit complex

Reply:
Thanks for the explanation @infoxicator